distribution/server/src/assemble/LICENSE.bin.txt

```diff
@@ -558,7 +558,6 @@ BSD 2-Clause License
 MIT License
  * Java SemVer -- com.github.zafarkhaja-java-semver-0.9.0.jar -- licenses/LICENSE-SemVer.txt
  * SLF4J -- licenses/LICENSE-SLF4J.txt
-    - org.slf4j-jul-to-slf4j-1.7.32.jar
     - org.slf4j-slf4j-api-1.7.32.jar
     - org.slf4j-jcl-over-slf4j-1.7.32.jar
```

Comment:
We need to redirect jcl-over-slf4j due to https://github.com/apache/pulsar/pull/1740 that it's an internal dependency to commons-logging and we should handle within Pulsar itself.